domain/schema/model.go

```diff
@@ -454,6 +455,64 @@ CREATE TABLE cloud_container (
 
 CREATE UNIQUE INDEX idx_cloud_container_net_node
 ON cloud_container (net_node_uuid);
+
+CREATE TABLE instance_data (
+    uuid            	   TEXT PRIMARY KEY,
+    machine_uuid    	   TEXT NOT NULL,
+    instance_id 	   TEXT NOT NULL,
+    display_name 	   TEXT NOT NULL,
+    arch            	   TEXT,
+    mod 	    	   INT,
+    root_disk 	    	   INT,
+    root_disk_source       TEXT,
+    cpu_cores 		   INT,
+    cpu_power 		   INT,
+    availability_zone_uuid TEXT,
+    virt_type 		   TEXT,
+    CONSTRAINT             fk_machine_machine_uuid
+        FOREIGN KEY            (machine_uuid)
+        REFERENCES             machine(uuid),
+    CONSTRAINT             fk_availability_zone_availability_zone_uuid
+        FOREIGN KEY            (availability_zone_uuid)
+        REFERENCES             availability_zone(uuid)
+);
+
+CREATE TABLE instance_tag (
+    uuid            TEXT PRIMARY KEY,
+    tag 	    TEXT NOT NULL
+);
+
+CREATE TABLE instance_tag_reference (
+    uuid            TEXT PRIMARY KEY,
+    tag_uuid 	    TEXT NOT NULL,
+    instance_uuid   TEXT NOT NULL,
+    CONSTRAINT      fk_instance_tag_tag_uuid
+        FOREIGN KEY     (tag_uuid)
+        REFERENCES      instance_tag(uuid),
+    CONSTRAINT      fk_instance_data_instance_uuid
+        FOREIGN KEY     (instance_uuid)
+        REFERENCES      instance_data(uuid)
+);
+
+CREATE TABLE lxd_profile (
+    uuid            TEXT PRIMARY KEY,
+    profile_id 	    TEXT NOT NULL
```

Comment:
yeah, I think this is the name of the profile. I'm not sure because in the legacy state we just have a slice of strings as charm profiles. Should I simply change it to `name`?

Reply:
Yeah, I think so. Are you envisaging the profile blob to be stored on this table?

```sql
CREATE TABLE lxd_profile (
    uuid            TEXT PRIMARY KEY,
    name 	        TEXT NOT NULL,
    -- Denormalized lxd profile blob.
    profile_blob    TEXT NOT NULL
)
```

Reply:
I wasn't going to add the blob on this PR. Actually I only created the lxd_profile table to hold the profile names, since it's the only thing we need for the charm profiles. I know that later the lxd_profile table will hold everything else we need about the profiles elsewhere.

Reply:
Done, renamed to `name`.